### DISCUSSION TITLE
Add Neovim-specific Lua setup

### DISCUSSION THREAD

**Discussion by mizlan** [mcchrish/nnn.vim#98] [Ideas]
Request to add an API like the following:

```lua
require('nnn-vim').setup{
  layout = { left = '20%' },
  session = 'local'
}
```

This would only work for Neovim and would not affect Vim.

**Comment by jarun** [comment]
Please go ahead.

**Comment by mizlan** [comment]
Old news but this is done.

**Reply by mizlan** [reply]
I can't figure out how to close this discussion thread... how unfortunate.

**Reply by jarun** [reply]
Me neither. Just leave it be.

**Reply by mcchrish** [reply]
One of the most requested features in github discussion  https://github.com/github/feedback/discussions/3097
Hopefully it will be implemented soon. For now let's leave it.
